packages/loader/README.md

```diff
@@ -0,0 +1,41 @@
+# loader
+
+purely makes use of css to provide a ripple effect.
+
+### Installation
+
+> yarn add @onaio/loader
+
+### customization(props)
+
+The customization object interface is as defined below.
+
+```js
+{
+  borderColor: string;
+  borderStyle: string;
+  borderWidth: string;
+  height: string;
+  minHeight: string;
+  width: string;
+}
+```
+
+#### Code example
+
+Relative to the default setting:
+
+- sets ripple effect color to purple-ish
+- changes the border style
+
+```javascript
+import Loader from '@onaio/loader'
+
+const loaderStyle = {
```

Comment:
Lets name this variable something like `props` so that it is more clear these are props